CHANGES.rst

```diff
@@ -64,6 +68,8 @@ Cubeviz
 
 - Fixed copious warnings from spaxel tool when data has INF. [#3368]
 
+- Cubeviz-specific helper-level methods are deprecated and will be removed in the future in favor of plugin APIs. [#3388]
```

Comment:
```suggestion
- Cubeviz-specific helper-level methods are deprecated and will be removed in the future in favor of plugin APIs as configs are centralized. [#3388]
```

Reply:
Does Imviz also need an entry?

Reply:
the only thing I deprecated for Imviz was `get_aperture_photometry_results`... I guess I'll just point that out specifically